wayshot/config.toml

```diff
@@ -0,0 +1,23 @@
+# base screenshot properties
+[base]
+# output to take screenshot
+# output = "default"
+# should contain cursor?
+cursor = false
+# should copy screenshot to clipborad?
+clipboard = false
+# should write screenshot as file in filesystem?
+file = true
+# should write screenshot in stdout?
+stdout = false
```

Comment:
I disagree that we should be writing to all of these at once. There should only be one output. For example the screenshot tools on MacOS and Windows (and other Linux apps) also only allow saving to 1 location. I myself would be confused if it saved to a file and to my clipboard.

As my comment on #100 and a comment on this PR from me elsewhere, I would want to avoid overloading this PR with extra features. Let's first start with adding a config, then we can change behaviour.

Reply:
I cannot agree with you on this problem. All the software that I have used (`flameshot`, `sharex`, `watershot`) does this. Hence, I wouldn't even use the software that doesn't allow me do that. IMO it's an essential thing to do, especially considering how easy it's done and that's no additional burden for this feature is needed.
I always take screenshot in both `clipboard` and a `file` outputs. I share a screenshot, or a part of it in some chats and can review the backup file in any point of time I want on my drive.
The `stdout` is no different. If user wants to have screenshot data in both `stdout` and a `file` backup - why wouldn't let user do that? Remember, that it's a switch, not a forced feature.
The reasons to have multiple backups for screenshots are enough. Maybe the `clipboard` is broken and the screenshot is not saved anywhere? Maybe it's `stdout` being broken? The screenshot data itself could be critical to save at the moment of time (e.g. some screenshot of some stream that's not recorded anywhere, or a moment in online game that you won't be able to reproduce ever again?), and just throwing an error would be very annoying and absolutely not helpful from user perspective.